module-system/sov-modules-core/src/runtime/capabilities.rs

```diff
@@ -1,15 +1,44 @@
 #![deny(missing_docs)]
-
 //! The rollup capabilities module defines "capabilities" that rollup must
 //! provide if they wish to use the standard app template.
 //! If you don't want to provide these capabilities,
 //! you can bypass the Sovereign module-system completely
 //! and write a state transition function from scratch.
 //! [See here for docs](https://github.com/Sovereign-Labs/sovereign-sdk/blob/nightly/examples/demo-stf/README.md)
 
-use sov_modules_core::{Context, WorkingSet};
 use sov_rollup_interface::da::{BlobReaderTrait, DaSpec};
 
+use crate::{Context, WorkingSet};
+
+/// The kernel is responsible for managing the inputs to the `apply_blob` method.
```

Comment:
Are these comments outdated? It seems like the Kernel and BlobSelector functionality is separated and the power of "reorder blobs" is more appropriate for BlobSelector?